content/docs/studio/view-settings.md

```diff
@@ -0,0 +1,92 @@
+# Additional Settings for a View
+
+If you are creating a view for a DVC repo, and if the DVC repo is at the root of
+the Git repository and does not reference remote/cloud storage, then you can
+successfully visualize it without specifying additional settings.
+
+In other cases, you could be trying to create views from non-DVC repositories.
+Or, your project may be in a sub-directory inside a monorepo. Or, the data to be
+visualized could be in custom files in your repository or even in remote / cloud
+storages. In all these scenarios, you will need to specify advanced settings so
+that DVC Studio is able to access the data required for visualization. These
+scenarios have been explained in details below.
+
+## Non-DVC repositories
+
+DVC Studio creates views by identifying datasets, metrics and hyperparameters
+defined in your Git repositories. These details (datasets, metrics and
+hyperparameters) are stored in your Git repositories as CSV, JSON or YAML files.
+You can add these details to your Git repositories in two ways:
+
+1. **Set up DVC repositories**: You can use [DVC](https://dvc.org/) and Git to
+   version your code, data and models all within your Git repositories. By using
+   DVC, you can be sure not to bloat your repositories with large volumes of
+   data or huge models. These large assets reside in cloud or other remote
+   storage locations. You will simply track their version info in Git.
+
+   Refer to the [DVC documentation](https://dvc.org/doc) to initialize a DVC
+   repository. You can then connect to this DVC repository and create a view as
+   described [earlier](/doc/studio/create-view). DVC Studio automatically
+   detects metrics, plots, and hyperparameters files specified in the project's
+   `dvc.yaml`. And every time you push a new commit to this DVC repository, your
+   view will reflect the new changes.
+
+2. **Specify custom files with your metrics and parameters**: If you are working
+   with a non-DVC repository, you can still create views for it if your
+   repository contains CSV, JSON or YAML files with the metrics and
+   hyperparameters that you want to visualize. To enable DVC Studio to visualize
+   such custom data, simply
+   [specify the custom files](#specifying-view-settings) with your metrics and
+   parameters, and DVC Studio will efficiently generate tables and plots for
+   your custom input. For instance, if you have an ML project for which you
+   generate and save metrics either manually or using some ML tracking tools,
+   then you can create a view for this project by specifying the file (within
+   your Git repo) which contains your saved metrics.
```

Comment:
```suggestion
   with a non-DVC repository, you can still create views for it provided that
   metrics and hyperparameters are stored in CSV, JSON or YAML files. To
   visualize such custom data, simply
   [specify the custom files](#specifying-view-settings) to use, and DVC Studio
   will efficiently generate tables and plots for your custom input. For instance,
   if you have an ML project for which you generate and save metrics either manually
   or using some ML tracking tools, then you can create a view for this project by
   specifying the file (within your Git repo) which contains your saved metrics.
```

Reply:
or even briefer:

```suggestion
   with a non-DVC repository, you can still create views for it provided that
   metrics and hyperparameters are stored in CSV, JSON or YAML files. To
   visualize such custom data, simply
   [specify the custom files](#specifying-view-settings) to use, and DVC Studio
   will efficiently generate tables and plots for your custom input.
```

Reply:
I think the extra explanation may help some people. So, accepting your earlier suggestion.